### DISCUSSION TITLE
Can I get a domain without the website being finished yet?

### DISCUSSION THREAD

**Discussion by TalinTheDev** [is-a-dev/register#5541] [General]
I'm a middle school student and don't get always get a lot of free time. We just finished testing for high schools so I hadn't been able to work on my website at all. I know I will definitely have one so it's not like I'm reserving it and just stopping others from using it but I just need time before the website is ready. Can I get the domain right now?

For reference, I already forked and made the entry in `/domains`. Here is my file: [talinsharma.json](https://github.com/TalinTheDev/register/commit/cfb89ada6f3d0894d918add8a95f45bef14588f5)

This is the the repo my website will be on: [talinsharma](https://github.com/TalinTheDev/talinsharma)

And it's going to be hosted on railway btw.

**Comment by wdhdev** [comment]
It is a requirement to complete your website, but I'll allow your PR. Just make sure to complete your website 😃
